design-proposals/new-proposals-framework.md

```diff
@@ -0,0 +1,72 @@
+# Overview
+
+This proposal outlines a new process for managing KubeVirt Enhancement Proposals (VEPs), emphasizing centralized prioritization and enhanced SIG involvement and collaboration. The proposed changes aim to focus the community's efforts on prioritized pull requests, increase the review bandwidth, and ensure clear visibility of feature progress and associated issues.
+
+## Motivation
+
+KubeVirt’s current approach to proposals is unstructured and has outgrown the community repo, making it difficult to adequately manage both the repo and the design proposals. The current process does not provide a way to prioritize development and review efforts during the development cycle, nor is there a way to derive a roadmap for a specific release. The process also lacks any formal SIG involvement. Being in the community repo also makes it difficult to grow the reviewer and approver list as there are conflicting purposes.
+
+
+With this proposal, we aim to:
+
+- Optimize contributors' and reviewers' bandwidth and experience.
+- Strengthen SIG ownership and accountability.
+- Improve the alignment of enhancements with the project's goals and architecture.
+- Separate the enhancement process from community matters.
+
+# Goals
+
+1. Introduce a new repository, `kubevirt/enhancements`, to house all VEPs.
+2. Prioritize VEPs at the start of each development cycle to streamline review and focus efforts.
+3. Establish SIG-specific review processes with a dedicated reviewer for each VEP.
+4. Create issues to track the progress, maturity, and associated bugs for each accepted VEP.
+5. Establish ownership of the VEP and any additional SIG collaboration.
+
+## Non-Goals
+
+- Removing the central approval mechanism at this stage.
```

Comment:
Out of curiosity, why not remove the approvals from the community repository right away? Maybe we could consider making the transition in phases, though I'm not sure exactly how that would work.